spec/android_unit_test_checker_spec.rb

```diff
@@ -0,0 +1,286 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module Danger
+  describe Danger::AndroidUnitTestChecker do
+    it 'is a plugin' do
+      expect(described_class.new(nil)).to be_a Danger::Plugin
+    end
+
+    describe 'with Dangerfile' do
+      before do
+        @dangerfile = testing_dangerfile
+        @plugin = @dangerfile.android_unit_test_checker
+
+        allow(@plugin.github).to receive(:pr_labels).and_return(['my_label'])
+
+        stub_const('GitDiffStruct', Struct.new(:type, :path, :patch))
+      end
+
+      it 'shows the right number of errors when a PR adding new classes that do not have corresponding tests' do
```

Comment:
With your latest changes, this doesn't just check the _number_ of errors anymore, but the actual errors now
```suggestion
      it 'shows the right errors when a PR adds new classes that do not have corresponding tests' do
```